### ISSUE TITLE
Without Post Processing fog in nebula is broken since #1846.

### ISSUE THREAD

**Issue by tomimaki** [scp-fs2open/fs2open.github.com#2227]
As is visible in topic below, objects far away from player are not covered by fog when post_process is off.
https://www.hard-light.net/forums/index.php?topic=96182.msg1891537#msg1891537
Before #1846
![screen0469](https://user-images.githubusercontent.com/18072803/71818411-2ce3c980-3089-11ea-9418-c1fcb697f8f4.png)
After #1846
![screen0470](https://user-images.githubusercontent.com/18072803/71818429-3c631280-3089-11ea-8cad-985556b37c9a.png)
Checked on Nvidia 1050 (441.66 drivers) and Intel (drivers from 2019-11-06)

**Comment by Baezon** [comment]
For posterity, I'll mention that this is caused by the `srgb_to_linear` conversion here https://github.com/scp-fs2open/fs2open.github.com/blob/master/code/def_files/data/effects/fog-f.sdr#L35. Without post processing, and that conversion removed it fogs to the correct color again. I've bought up only conditionally doing this step, but the other graphics coders seemed to prefer deprecating `-no_post_proc` entirely instead.

**Comment by JohnAFernandez** [comment]
Should probably stick a won't fix on this.

**Comment by MjnMixael** [comment]
Just checking in. If this is not going to be fixed then, given how critical the Nebula environment is to FS2 and many mods, the commandline flag should be deprecated and the Post Processing option should be removed.

**Comment by MjnMixael** [comment]
Based on the above conversation and the lack of activity here, I'm going to close as not planned. As always, feel free to re-open if that changes.
